rascaline-c-api/cmake/fetch-content-cmake-3.11/FetchContent.cmake

```diff
@@ -0,0 +1,916 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
```

Comment:
Is this appearing in our docs?

Reply:
Not sure what you mean? I took this file verbatim from a cmake-3.11.2 installation